grammar.js

```diff
@@ -289,48 +295,48 @@ module.exports = grammar({
 
     variable_declaration: $ => seq(
       field('type', $._type),
-      commaSep1($.variable_declarator)
+      commaSep1(field('variable_declarator', $.variable_declarator))
```

Comment:
This is in a `commaSep1`, shouldn't I instead remove the field name altogether based on your second suggestion?

Reply:
I think this is a slightly different case because there are also _other_ fields here. Since there's also the `type` field, having the field names helps grabbing the right subterms when querying.

On the other hand, for a production similar to

    $.foo = seq('(', commaSep1(field('bar', $.bar)), ')')

all the subterms will have field `bar`. Matching `(foo bar:(_)@bar)` and `(foo (_)@bar)` will have the same result. Adding the field name does not help in distinguishing different subterms of `foo`. In those cases, I suggest to omit it.

Hope that clarifies what I mean!